ui/src/virtualconsole/vcspeeddialproperties.cpp

```diff
@@ -270,6 +270,43 @@ void VCSpeedDialProperties::slotRemoveClicked()
         delete it.next();
 }
 
+void VCSpeedDialProperties::applySelectedFactorsToAllClicked()
+{
+    QList <QTreeWidgetItem*> selectedItems = m_tree->selectedItems();
+    if (selectedItems.isEmpty())
+        return;
+    QTreeWidgetItem* selected = m_tree->selectedItems().first();
+
+    // clear focus on the combobox to apply the selected value
```

Comment:
@siegmund42 
This now clears the focus on the open combobox before the factors are copied.
Let me know if there is a more elegant way in QT to do this (i'm a qt noob)...